GEOSldas_App/ldas_setup

```diff
@@ -1079,7 +1092,7 @@ class LDASsetup:
 
         # DEFAULT rc files
         default_rc = glob.glob(etcdir+'/GEOSldas_*.rc')
-        assert len(default_rc)==4
+        assert len(default_rc)==6
```

Comment:
Does this change work for a stand-alone (offline) config of LDAS?  It's unclear to me how we can just change from 4 to 6 without breaking something for the offline system.  I'm probably misunderstanding this

Reply:
we added 2 more GEOSldas_*.rc in GEOSldas_App/,  so "build "( build from adas or stand-alone ldas)  puts 6 of them in install*/etc/.    Setup will take the det and ens ones if it is coupled run,  if it is stand-alone the setup will just take the GEOSldas_hist.rc.

Reply:
Ok, thanks